Detectors/TPC/qc/include/TPCQC/Clusters.h

```diff
@@ -49,9 +49,9 @@ class Clusters
 
   void fillADCValue(int cru, int rowInSector, int padInRow, int timeBin, float adcValue);
 
-  void normalize();
+  void normalize(const float nHBFPerTF = 128);
 
-  inline void analyse() { Clusters::normalize(); }
+  inline void analyse() { Clusters::normalize(); } // deprecated
```

Comment:
Might be used in macros, so I would not call it deprecated.
```suggestion
  inline void analyse() { Clusters::normalize(); }
```